### PR TITLE
Added alt attributes to images for better lighthouse score

### PR DESCRIPTION
**Pull request by calebnance**
Added alt attributes to images (also the nunjucks file had a lot of spaces at line end, so fixed those as well).

The alt text added in template is: `alt="{{language.value}} example of {{title}} in {{theme}} mode"`

So for HTML light mode example will read: `alt="html example of Anonymous Pro in light mode"`

This bumps up the accessibility score:
coding-fonts-accessibility-1

### CODE REVIEW

**Comment by calebnance** [comment]
also the lighthouse SEO score goes up from 80 => 90

File: src/_includes/font-page.njk [hunk 1]
```diff
@@ -3,21 +3,21 @@ layout: base
 ---
 
 {%- for theme in samples.themes -%}
-  <input type="radio"
```
Comment (chriscoyier) [inline src/_includes/font-page.njk:6]: I like the upgrade here! But can we clean up the noise in the changes here?

Comment (calebnance) [reply src/_includes/font-page.njk:6]: yes, no problem.

just to confirm you are talking about adding back the spaces at the end of some of the lines in this file? The cleanup i was doing was removing those extra spaces after line endings, etc. See below, that's all that is being removed/cleanedup.

Screen Shot 2020-12-04 at 10 28 59 AM

**Comment by chriscoyier** [comment]
It's still registering those lines as changed. Meh. I dunno. Not a huge deal it just makes it harder to understand the nature of the exact change.

**Comment by chriscoyier** [comment]
Very good move having the proper alt text

**Pull request merged**